crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -313,6 +326,11 @@ impl<'a> InvariantExecutor<'a> {
                         TestCaseError::fail(format!("Could not make raw evm call: {e}"))
                     })?;
 
+                // Collect and merge coverage if `forge snapshot` context.
+                if is_coverage_context() {
+                    invariant_test.merge_coverage(call_result.coverage.clone());
+                }
+
```

Comment:
There is no need in checking if we're in coverage context as `call_result.coverage` will always be `None` if coverage inspector is not configured

Reply:
ah, missed that, thank you, will remove!

Reply:
removed in https://github.com/foundry-rs/foundry/pull/8265/commits/78104bcc16250c0069cc60cf59bf97f51c5513a2